src/System.Runtime/tests/System/Reflection/ModuleTests.cs

```diff
@@ -80,15 +81,9 @@ public void CustomAttributes()
         }
 
         [Fact]
-        public void FullyQualifiedName()
```

Comment:
Generally I find it very odd that we are deleting tests - if they pass for coreclr, why not disable them just for uapaot?